terraform/workspace_variables/production.tfvars

```diff
@@ -0,0 +1,27 @@
+#PaaS
+cf_space               = "bat-prod"
+paas_app_environment   = "prod"
+paas_web_app_host_name = "www"
+paas_web_app_instances = 1
```

Comment:
We should have at least 2 in prod. Or is it temporary?

Reply:
yes temporary, I'll change it later, as its causing me rebase issues for the next PR